### PR TITLE
[TT-4027] document certs APIs

### PR DESCRIPTION
**Pull request by jeffy-mathew**
this PR documents `/tyk/certs` API

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -3004,6 +3112,154 @@ components:
...
+      properties:
+        certs:
+          items:
+            type: string
+          type: array
+          x-go-name: Certs
+      type: object
+      x-go-package: github.com/TykTechnologies/tyk
+    APIAllCertificateBasics:
+      description: APIAllCertificates represents a list of certificates
```
Comment (vaske) [inline swagger.yml:3126]: description is same as `APIAllCertificates`, did you forget to put here `APIAllCertificateBasics`?

**Review by vaske — APPROVED** [review]
Overall looks good, I left one minor comment

File: swagger.yml (hunk 1 above)
Comment (jeffy-mathew) [reply swagger.yml:3126]: typo, correcting it now :)

**Review by lghiur — APPROVED** [review]

**Pull request merged**